### PR TITLE
Add rule types severity

### PR DESCRIPTION
**Pull request by teodor-yanev**
Fixes https://github.com/stacklok/minder-rules-and-profiles/issues/64

### CODE REVIEW

**Comment by JAORMX** [comment]
Have you tested if upgrade of these rules work? I haven´t tried it out, but it would be worth doing it.

In a local env, do `minder ruletype create` with the rules before this change. Then do a `minder ruletype apply` with the change, and see that it all goes smoothly and the severity is indeed updated.

File: rule-types/github/invisible_characters_check.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 version: v1
 type: rule-type
 name: invisible_characters_check
+severity:
+  value: medium
```
Comment (JAORMX) [inline rule-types/github/invisible_characters_check.yaml:5]: High? Maybe

File: rule-types/github/license.yaml [hunk 2]
```diff
@@ -2,6 +2,8 @@
 version: v1
 type: rule-type
 name: license
+severity:
+  value: medium
```
Comment (JAORMX) [inline rule-types/github/license.yaml:6]: Low

File: rule-types/github/mixed_scripts_check.yaml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 version: v1
 type: rule-type
 name: mixed_scripts_check
+severity:
+  value: medium
```
Comment (JAORMX) [inline rule-types/github/mixed_scripts_check.yaml:5]: High? Maybe

File: rule-types/github/no_open_security_advisories.yaml [hunk 4]
```diff
@@ -2,6 +2,8 @@
 version: v1
 type: rule-type
 name: no_open_security_advisories
+severity:
+  value: medium
```
Comment (JAORMX) [inline rule-types/github/no_open_security_advisories.yaml:6]: Low

File: rule-types/github/repo_action_allow_list.yaml [hunk 5]
```diff
@@ -2,6 +2,8 @@
 version: v1
 type: rule-type
 name: repo_action_allow_list
+severity:
+  value: medium
```
Comment (JAORMX) [inline rule-types/github/repo_action_allow_list.yaml:6]: Info

File: rule-types/github/secret_push_protection.yaml [hunk 6]
```diff
@@ -2,6 +2,8 @@
 version: v1
 type: rule-type
 name: secret_push_protection
+severity:
+  value: medium
```
Comment (JAORMX) [inline rule-types/github/secret_push_protection.yaml:6]: High

File: rule-types/github/secret_scanning.yaml [hunk 7]
```diff
@@ -2,6 +2,8 @@
 version: v1
 type: rule-type
 name: secret_scanning
+severity:
+  value: medium
```
Comment (JAORMX) [inline rule-types/github/secret_scanning.yaml:6]: High

**Comment by teodor-yanev** [comment]
@JAORMX Thanks for your feedback! I updated the rules.

**Pull request merged**